microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -6,12 +6,10 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: pwa
-ms.date: 04/27/2023
+ms.date: 02/20/2024
 ---
 # Build PWAs for the sidebar in Microsoft Edge
 
-![Experimental feature](./sidebar-images/experimental-tag.png)
```

Comment:
Can we remove "Experimental feature" tag before it is standardized?

Reply:
not sure i understand the question, but I entered a PR Comment saying to remove the .png in this dir if this png has become unused.

i plan to inspect / Search in VS Code to confirm the png is not used, and then delete the .png file.  If Patrick doesn't do that first.

Reply:
The "Experimental feature" image was here because the feature needed to be enabled before being used. This is no longer the case. Developers can use the feature with Edge without having to ask users to enable it first.
So, in that sense, it's no longer experimental in Edge.

I understand your question being about the feature not being a web standard yet. That's true, but I think that's fine. We typically don't document standard web features on learn.microsoft.com (LMC) because we already document them on MDN. What we write about on LMC is specific to Edge only. We could add a quick note about the fact that this feature is not supported across browsers if needed. But, I'm tempted to leave it as is, mostly because of the name of the manifest member being `edge_side_panel`. And also because adding the manifest will not break other browsers. It will just silently be ignored.

Reply:
removed file experimental-tag.png